src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -15,6 +15,8 @@ internal static class SecurityProtocol
         public const SslProtocols DefaultSecurityProtocols =
             SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
 
+        public const SslProtocols SystemDefaultSecurityProtocols = SslProtocols.None;
```

Comment:
> If the protocols are not specified by the application you should let libcurl do whatever it defaults to. 

Ok.  It doesn't look like you're touching System.Net.Http as part of this change, but whoever updates WinHttpHandler to respect the new setting should just do the same for CurlHandler at the same time.